src/L2VoteAggregator.sol

```diff
@@ -67,12 +82,16 @@ abstract contract L2VoteAggregator {
   mapping(uint256 proposalId => ProposalVote) public proposalVotes;
 
   /// @dev Emitted when a vote is cast on L2.
-  event VoteCast(address indexed voter, uint256 proposalId, VoteType support, uint256 weight);
+  event VoteCast(
+    address indexed voter, uint256 proposalId, VoteType support, uint256 weight, string reason
+  );
 
   /// @param _votingToken The token used to vote on proposals.
   /// @param _governorMetadata The `GovernorMetadata` contract that provides proposal information.
   /// @param _l1BlockAddress The address of the L1Block contract.
-  constructor(address _votingToken, address _governorMetadata, address _l1BlockAddress) {
+  constructor(address _votingToken, address _governorMetadata, address _l1BlockAddress)
+    EIP712("L2VoteAggregator", "1")
```

Comment:
Is this okay to hardcode? It seemed fine from the EIP.

Reply:
I think we're good here to hardcode. re: @apbendi, in this case, the 2nd constructor arg is simply "version." stuff like `chainId`, contract `address`, etc is all done in the oz EIP712 constructor. See: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/cryptography/EIP712.sol#L66-L75

Reply:
Ahh ok, yes. Sorry, I misread this and thought it was the chainId. Agree—good to hardcode!